### ISSUE TITLE
Issue found on page 'Text Functions': bar output contains '?' after bar output, but what does that mean? 

### ISSUE THREAD

**Issue by rpbouman** [duckdb/duckdb-web#3301]
The function bar is documented here:

https://duckdb.org/docs/sql/functions/char#barx-min-max-width

The given example works as advertised, but when I use it on my data I often but not always see the bar output is followed by '?'.
I am not sure what this means and that is not documented.

I suspect it has something to do with the output being an approximate indicator of the value such as when the value cannot be expressed exactly by a discrete number of bars. 

For example when I do:

`select bar(5.5, 0, 20, 10);`

I get 

```
┌─────────────────────┐
│ bar(5.5, 0, 20, 10) │
│       varchar       │
├─────────────────────┤
│ ██?                 │
└─────────────────────┘
```

I'm fine with documenting this, but I'd need to know if my assumption is correct, and also what the name is of this feature

**Comment by Tishj** [comment]
I suspect that's a bug in the `bar` function

**Comment by rpbouman** [comment]
Hi @Tishj !

Ugh, I feel so silly now! You're absolutely right - I recently moved to win 11 and apparently the default for the command shell, or at least, the way I got the machine, is not unicode/utf8.

For whomever needs it, do;

"Time & Language > Language & Region > Administrative Language Settings'", then hit "Change system locale" and check "Beta: Use Unicode UTF8 for world-wide language support". (Reboot required).
